include/TrigScint/TestBeamClusterProducer.h

```diff
@@ -5,15 +5,15 @@
  */
 
 #ifndef TRIGSCINT_TESTBEAMCLUSTERPRODUCER_H
-#define TRIGSCINT__TESTBEAMCLUSTERPRODUCER_H
+#define TRIGSCINT_TESTBEAMCLUSTERPRODUCER_H
 
```

Comment:
This change I did forget to mention but clang did catch a minor bug here, in the previous version the include guard wasn't actually working. It didn't make a difference because the TB cluster producer header was never included outside of its corresponding source file but if someone would have included it again in some other file, it would have broken